### PR TITLE
[ElmSharp] Add more preloading components

### PR DESCRIPTION
**Pull request by JoonghyunCho**
### Description of Change ###
This PR enables two more preloading options on `PreloadedWindow`.
- Preloading widgets which are used when initializing Xamarin Forms Tizen platform renderers.
  - `Entry`
  - `Scroller`
  - `Box`
  - `Label`
  - `GenList`
  - `Button`
  - `Check`
  - `Naviframe`
  - `Slider`
  - `Spinner`
  - `ProgressBar`
  - `GestureLayer`
  - `Polygon`
  - `Image`

- Preloading events which are used at application launching time.
  - `PreloadedWindow.BackButtonPressed`

### API Changes ###
N/A

### CODE REVIEW

File: src/ElmSharp/ElmSharp/Elementary.cs [hunk 1]
```diff
@@ -207,7 +207,8 @@ public static FocusAutoScrollMode FocusAutoScrollMode
         /// <since_tizen> preview </since_tizen>
         public static void Initialize()
         {
-            Interop.Elementary.elm_init(0, null);
+            if (!Window.IsPreloaded)
```
Comment (rookiejava) [inline src/ElmSharp/ElmSharp/Elementary.cs]: Apply this to `ThemeOveraly()` as well.

Comment (JoonghyunCho) [reply src/ElmSharp/ElmSharp/Elementary.cs]: Ok!

File: src/ElmSharp/ElmSharp/PreloadedWindow.cs [hunk 2]
```diff
@@ -46,6 +49,29 @@ public void InitializeBaseLayout()
...
+            new GenList(this).Unrealize();
+            new Button(this).Unrealize();
+            new Check(this).Unrealize();
+            new Naviframe(this).Unrealize();
+            new Slider(this).Unrealize();
+            new Spinner(this).Unrealize();
+            new ProgressBar(this).Unrealize();
+            new GestureLayer(this).Unrealize();
+            new Polygon(this).Unrealize();
+            new Image(this).Unrealize();
```
Comment (JoonghyunCho) [inline src/ElmSharp/ElmSharp/PreloadedWindow.cs]: We need a default image source to load..

File: src/ElmSharp/ElmSharp/PreloadedWindow.cs [hunk 3]
```diff
@@ -25,6 +25,9 @@ public PreloadedWindow(bool useBaseLayout=true) : base("PreloadWindow")
             s_precreated = this;
             if (useBaseLayout)
                 InitializeBaseLayout();
+            CreateWidgets();
+            BackButtonPressed += DummyHandler;
```
Comment (myroot) [inline src/ElmSharp/ElmSharp/PreloadedWindow.cs]: How about use local function? 
https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/classes-and-structs/local-functions

Comment (JoonghyunCho) [reply src/ElmSharp/ElmSharp/PreloadedWindow.cs]: You mean to add/remove event in the method like `CreateWidgets()`? Ok. Sounds coherent.

Comment (JoonghyunCho) [reply src/ElmSharp/ElmSharp/PreloadedWindow.cs]: Done

File: src/ElmSharp/ElmSharp/PreloadedWindow.cs [hunk 4]
```diff
@@ -46,6 +50,25 @@ public void InitializeBaseLayout()
             conformant.SetContent(BaseLayout);
         }
 
+        public void CreateWidgets()
```
Comment (rookiejava) [inline src/ElmSharp/ElmSharp/PreloadedWindow.cs]: I don't think this method creates widgets. So, how do you feel about renaming? e.g. WarmupXXX , ...

Comment (JoonghyunCho) [reply src/ElmSharp/ElmSharp/PreloadedWindow.cs]: Agree. I think we can go with No.1. what do you think? 
1. WarmupWidgets()
2. PreloadWidgets()
3. PrepareWidgets()

Comment (rookiejava) [reply src/ElmSharp/ElmSharp/PreloadedWindow.cs]: Those are (whatever) better than previous one for me.

**Pull request merged**